classes/ProductPresenterFactory.php

```diff
@@ -66,6 +69,7 @@ public function getPresentationSettings()
         $settings->stock_management_enabled = Configuration::get('PS_STOCK_MANAGEMENT');
         $settings->showPrices = Configuration::showPrices();
         $settings->lastRemainingItems = Configuration::get('PS_LAST_QTIES');
+        $settings->showLabelOOSListingPages = (bool) Configuration::get('PS_SHOW_LABEL_OOS_LISTING_PAGES');
```

Comment:
It could be nice to use the configuration object here as you did in `src/Adapter/Presenter/Product/ProductLazyArray.php`, so that you can do : 
`$this->configuration->get('PS_WEIGHT_UNIT');`, 

but it's a little bit out of scope I guess.